client/src/modules/stock/depot-selection.modal.html

```diff
@@ -7,46 +7,41 @@
 
 <div class="modal-body">
 
-  <!-- link to depots page if depots not found -->
-  <p class="text-info">
-    <i class="fa fa-question-circle-o"></i>
-    <span translate>DEPOT.MODAL.NO_DEPOTS</span>
-
-    <a href ui-sref="depots.create">
-      <i class="fa fa-link"></i> <span translate>DEPOT.ADD_DEPOT</span>
-    </a>
-  </p>
-
-  <div uib-alert class="alert-warning" ng-if="$ctrl.hasNoDefaultDepot">
+  <div uib-alert class="alert-warning" ng-if="$ctrl.hasNoDefaultDepot && $ctrl.depots.length>0">
     <p>
       <strong> <i class="fa fa-warning"></i> <span translate>DEPOT.MODAL.NO_DEFAULT_DEPOT</span></strong>
       <br />
       <span translate>DEPOT.MODAL.SELECTION_EXPLANATION</span>
-      <strong><a ui-sref="depots.create"><b><span translate>DEPOT.ADD_DEPOT</span></b></a></strong>.
     </p>
   </div>
 
+  <div uib-alert class="alert-warning" ng-if="$ctrl.depots.length===0">
```

Comment:
Note that, if there are no depots in the enterprise, this will give a slightly misleading message.  There is also the problem that you cannot exit the modal if you trigger it anymore - the link used to be a hack to work around that.  We should probably just fix the origin of the problem them - presenting links only to people who can use them.

Reply:
in that case I can just correct the message to help the user to know that he has no permission on a depot or there is no depot enregistered yet, he can cantact the administrator

Reply:
That sounds like a good idea :)